frontend_lib/src/helper.js

```diff
@@ -18,6 +18,7 @@ import {
   getReservedUsernames,
   getUsernameAvailability
 } from './action.async.js'
+import { MENTION_CONSTANT } from './mentionOrLink'
```

Comment:
missing .jsx

Reply:
Fixed in [9d13cc1](https://github.com/tracim/tracim/pull/6518/commits/9d13cc12cbc7970243cb0b47d750df5853fde60a)